src/templates/playgroundCreateCollectionTemplate.ts

```diff
@@ -25,9 +25,19 @@ db.createCollection(collection);
     viewOn: <string>,
     pipeline: <pipeline>,
     collation: <document>,
-    writeConcern: <document>
+    writeConcern: <document>,
+    timeseries: { // Added in MongoDB 5.0
+      timeField: <string>, // required for time series collections
+      metaField: <string>,
+      granularity: <string>
+    },
+    expireAfterSeconds: <number>,
+    clusteredIndex: <document>, // Added in MongoDB 5.3
```

Comment:
I see for timeseries we list some options but for clusteredIndex and a few others we just say `<document>`. Wondering if we should put more stuff in there? Maybe best like this so they have to read the documentation.

Reply:
We're following the docs there: https://www.mongodb.com/docs/v5.3/reference/method/db.createCollection/
Which also only expands the time-series document. I'm not totally sure on the reasoning there. I think it's alright though, since these are more complex configurations I think expanding all of them might make us have to update this more frequently, and could possibly lead some users astray on differing mongodb server versions.